planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/util/YAMLmodulesOptimizerParser.java

```diff
@@ -357,24 +356,42 @@ private static QualityInformation parseQoS(Map<String, Object> qosInformation) {
 
    public static boolean getScalabilityCapabilitiesOfModule(Map<String, Object> modules, String elementName) {
       try {
-         Map<String, Object> moduleInfo = (Map<String, Object>) modules.get(elementName);
-         Map<String, Object> moduleProperties = (Map<String, Object>) moduleInfo
-               .get(TOSCAkeywords.MODULE_PROPERTIES_TAG);
-         boolean moduleCanScale = (boolean) moduleProperties.get(TOSCAkeywords.MODULE_AUTOSCALE_PROPERTY);
-         return moduleCanScale;
-      } catch (Exception e) {
+         Map<String, Object> moduleInfo;
+         if (modules.containsKey(elementName)) {
+            moduleInfo = (Map<String, Object>) modules.get(elementName);
+         } else {
+            // not specified the scalability. Default value for not specified is
+            // true;
+            return true;
```

Comment:
@kiuby88 Thank you for the review. 
I've addressed all your comments and pushed. 
Now the default option to scale a module is false.